### PR TITLE
[LLM] Add more transformers_int4 examples (MPT)

### PR DESCRIPTION
**Pull request by Oscilloscope98**
## Description

Add more transformers int4 examples

### 1. Why the change?

Add more model-specific examples for better user support. Add MPT for this initial PR

### 2. User API changes

N/A

### 3. Summary of the change 

Added model:
- [x] MPT

transformers_int4_pipeline_readme.md and transformers_int4_pipeline.py will be removed after model chatglm is added.

### 4. How to test?
- [x] Local test on Linux
- [x] Local test on Windows

### CODE REVIEW

File: python/llm/example/transformers/transformers_int4/mpt/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# MPT
```
Comment (jason-dai) [inline python/llm/example/transformers/transformers_int4/mpt/README.md:1]: 1. Install
2. Config
3. Run
2.1 Client
2.2 Server

Comment (Oscilloscope98) [reply python/llm/example/transformers/transformers_int4/mpt/README.md:1]: Have adjusted the structure as required :)

File: python/llm/example/transformers/transformers_int4/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# BigDL-LLM Transformers INT4 Optimization for Large Language Model
+BigDL-LLM supports INT4 optimizations to any Hugging Face Transformers models.
+
+In this directory, we provide several model-specific examples with BigDL-LLM INT4 optimizations. You can navigate to the folder corresponding to the model you want to use.
+
+We are adding more and more model-specific examples.
```
[the remark this answers is not in the document]
Comment (Oscilloscope98) [reply python/llm/example/transformers/transformers_int4/README.md]: Have updated. Thanks  :)

**Review by plusbang — APPROVED** [review]

**Comment by Oscilloscope98** [comment]
@jason-dai 
Merged for now, will raise other PRs to fix for any further comments :)

**Pull request merged**